### PR TITLE
[claroty_ctd] Added double new lines between the navigation links on the dashboard.

### PR DESCRIPTION
**Pull request by muskan-agarwal26**
Type of change
- Enhancement

## Proposed commit message

- The formatting of the dashboard links in the `Table of Content` section was not displaying correctly in versions above 8.14.0 Specifically, new links were appearing in a continuous paragraph rather than on separate lines. This issue has been resolved by adding double new lines in between.

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [ ] I have verified that all data streams collect metrics or logs.
- [ ] I have added an entry to my package's `changelog.yml` file.
- [ ] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).
- [ ] I have verified that any added dashboard complies with Kibana's [Dashboard good practices](https://docs.elastic.dev/ux-guidelines/data-viz/dashboard-good-practices) 

## How to test this PR locally

- Clone integrations repo.
- Install the elastic package locally.
- Start the elastic stack using the elastic package.
- Move to integrations/packages/claroty_ctd directory.
- Run the following command to run tests.

> elastic-package test
```
elastic-package test
2024/11/20 12:08:17  INFO New version is available - v0.107.1. Download from: https://github.com/elastic/elastic-package/releases/tag/v0.107.1
Run asset tests for the package
2024/11/20 12:08:17  INFO License text found in "/root/github/integrations/LICENSE.txt" will be included in package
--- Test results for package: claroty_ctd - START ---
╭─────────────┬─────────────┬───────────┬──────────────────────────────────────────────────────────────────────┬────────┬──────────────╮
│ PACKAGE     │ DATA STREAM │ TEST TYPE │ TEST NAME                                                            │ RESULT │ TIME ELAPSED │
├─────────────┼─────────────┼───────────┼──────────────────────────────────────────────────────────────────────┼────────┼──────────────┤
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-0f0d9db1-dbf3-4d2f-b87c-c5d75e6ea62d is loaded │ PASS   │      1.352µs │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-1051aba6-db32-473c-a24d-3b9200f22f17 is loaded │ PASS   │        244ns │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-3ff35f11-fb3c-40b1-bb47-a0d5345fc0c7 is loaded │ PASS   │        210ns │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-66d8c2cc-eeb7-420e-8c83-9c0cae66a7fe is loaded │ PASS   │        267ns │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-a91b7c98-8dd4-479e-bc35-71641c85db97 is loaded │ PASS   │        237ns │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-b7e0ad66-86c3-4b72-ae19-0576ed9fb1d6 is loaded │ PASS   │        253ns │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-ce5edf6b-6671-4d1e-adc0-834a76ff9169 is loaded │ PASS   │        271ns │
│ claroty_ctd │             │ asset     │ dashboard claroty_ctd-e2106b06-3d16-4014-9343-f63ebb90a114 is loaded │ PASS   │        290ns │
│ claroty_ctd │             │ asset     │ search claroty_ctd-0cab445c-f2f9-4549-8224-2c66b114e973 is loaded    │ PASS   │        207ns │
│ claroty_ctd │             │ asset     │ search claroty_ctd-4c469790-a269-4d9f-8616-816379cbc3ad is loaded    │ PASS   │        223ns │
│ claroty_ctd │             │ asset     │ search claroty_ctd-9334e7c2-86b7-4aa4-8b0e-26b52c58bfd0 is loaded    │ PASS   │        246ns │
│ claroty_ctd │             │ asset     │ search claroty_ctd-afa635ef-ec61-44f6-8eb3-c61332a6f987 is loaded    │ PASS   │        237ns │
│ claroty_ctd │             │ asset     │ search claroty_ctd-d7b4f221-71ef-4c93-b875-d03cd10a6743 is loaded    │ PASS   │        254ns │
│ claroty_ctd │ asset       │ asset     │ index_template logs-claroty_ctd.asset is loaded                      │ PASS   │        174ns │
│ claroty_ctd │ asset       │ asset     │ ingest_pipeline logs-claroty_ctd.asset-0.1.3 is loaded               │ PASS   │        183ns │
│ claroty_ctd │ baseline    │ asset     │ index_template logs-claroty_ctd.baseline is loaded                   │ PASS   │        176ns │
│ claroty_ctd │ baseline    │ asset     │ ingest_pipeline logs-claroty_ctd.baseline-0.1.3 is loaded            │ PASS   │        160ns │
│ claroty_ctd │ event       │ asset     │ index_template logs-claroty_ctd.event is loaded                      │ PASS   │        232ns │
│ claroty_ctd │ event       │ asset     │ ingest_pipeline logs-claroty_ctd.event-0.1.3 is loaded               │ PASS   │        190ns │
╰─────────────┴─────────────┴───────────┴──────────────────────────────────────────────────────────────────────┴────────┴──────────────╯
--- Test results for package: claroty_ctd - END   ---
Done
Run pipeline tests for the package
--- Test results for package: claroty_ctd - START ---
╭─────────────┬─────────────┬───────────┬──────────────────────────────────────────────┬────────┬──────────────╮
│ PACKAGE     │ DATA STREAM │ TEST TYPE │ TEST NAME                                    │ RESULT │ TIME ELAPSED │
├─────────────┼─────────────┼───────────┼──────────────────────────────────────────────┼────────┼──────────────┤
│ claroty_ctd │ asset       │ pipeline  │ (ingest pipeline warnings test-asset.log)    │ PASS   │ 598.132665ms │
│ claroty_ctd │ asset       │ pipeline  │ test-asset.log                               │ PASS   │  518.88892ms │
│ claroty_ctd │ baseline    │ pipeline  │ (ingest pipeline warnings test-baseline.log) │ PASS   │ 497.654599ms │
│ claroty_ctd │ baseline    │ pipeline  │ test-baseline.log                            │ PASS   │ 265.081566ms │
│ claroty_ctd │ event       │ pipeline  │ (ingest pipeline warnings test-event.json)   │ PASS   │ 487.214698ms │
│ claroty_ctd │ event       │ pipeline  │ test-event.json                              │ PASS   │ 383.290853ms │
╰─────────────┴─────────────┴───────────┴──────────────────────────────────────────────┴────────┴──────────────╯
--- Test results for package: claroty_ctd - END   ---
Done
Run policy tests for the package
--- Test results for package: claroty_ctd - START ---
No test results
--- Test results for package: claroty_ctd - END   ---
Done
Run static tests for the package
--- Test results for package: claroty_ctd - START ---
╭─────────────┬─────────────┬───────────┬──────────────────────────┬────────┬──────────────╮
│ PACKAGE     │ DATA STREAM │ TEST TYPE │ TEST NAME                │ RESULT │ TIME ELAPSED │
├─────────────┼─────────────┼───────────┼──────────────────────────┼────────┼──────────────┤
│ claroty_ctd │ asset       │ static    │ Verify sample_event.json │ PASS   │ 150.316791ms │
│ claroty_ctd │ baseline    │ static    │ Verify sample_event.json │ PASS   │ 143.021267ms │
│ claroty_ctd │ event       │ static    │ Verify sample_event.json │ PASS   │ 194.013365ms │
╰─────────────┴─────────────┴───────────┴──────────────────────────┴────────┴──────────────╯
--- Test results for package: claroty_ctd - END   ---
Done
Run system tests for the package
2024/11/20 12:08:27  INFO License text found in "/root/github/integrations/LICENSE.txt" will be included in package
2024/11/20 12:09:22  INFO Write container logs to file: /root/github/integrations/build/container-logs/claroty-ctd-cel-1732084762007023894.log
2024/11/20 12:10:22  INFO Write container logs to file: /root/github/integrations/build/container-logs/claroty-ctd-cel-1732084822713430982.log
2024/11/20 12:11:26  INFO Write container logs to file: /root/github/integrations/build/container-logs/claroty-ctd-tcp-1732084886811187191.log
2024/11/20 12:12:32  INFO Write container logs to file: /root/github/integrations/build/container-logs/claroty-ctd-udp-1732084952824760586.log
--- Test results for package: claroty_ctd - START ---
╭─────────────┬─────────────┬───────────┬───────────┬────────┬───────────────╮
│ PACKAGE     │ DATA STREAM │ TEST TYPE │ TEST NAME │ RESULT │  TIME ELAPSED │
├─────────────┼─────────────┼───────────┼───────────┼────────┼───────────────┤
│ claroty_ctd │ asset       │ system    │ asset     │ PASS   │ 41.976922326s │
│ claroty_ctd │ baseline    │ system    │ baseline  │ PASS   │ 41.740678336s │
│ claroty_ctd │ event       │ system    │ tcp       │ PASS   │ 44.413180221s │
│ claroty_ctd │ event       │ system    │ udp       │ PASS   │ 46.436344844s │
╰─────────────┴─────────────┴───────────┴───────────┴────────┴───────────────╯
--- Test results for package: claroty_ctd - END   ---
Done
``` 

## Screenshots
before
after

### CODE REVIEW

**Review by efd6 — COMMENTED** [review]
The issue does not have an issue associated with it, so there should be a description of the fault and what is being done to fix it in the proposed commit message; "Fix dashboard link issue in Kibana versions above 8.13.0." does not explain what is being fixed.

**Review by kcreddy — COMMENTED** [review]
@muskan-agarwal26, 

> The formatting of the dashboard links in the Table of Content section was not displaying correctly in versions above 8.14.0

Could you add screenshot of sample dashboard to the PR description of how the display was before (incorrect) and after (corrected)?

**Comment by efd6** [comment]
e.g.
Screenshot from 2024-11-21 16-52-00

MD expects double new lines for a `<br>`. I don't think we need the `<ul>` or the use of bold though. Suggest instead e.g.
`"markdown": "## Navigation\n\n[Overview(This Page)](/app/dashboards#/view/claroty_ctd-1051aba6-db32-473c-a24d-3b9200f22f17)\n\n[Alerts](/app/dashboards#/view/claroty_ctd-a91b7c98-8dd4-479e-bc35-71641c85db97)\n\n[Activity Log](/app/dashboards#/view/claroty_ctd-0f0d9db1-dbf3-4d2f-b87c-c5d75e6ea62d)\n\n[Asset](/app/dashboards#/view/claroty_ctd-ce5edf6b-6671-4d1e-adc0-834a76ff9169)\n\n[Baseline](/app/dashboards#/view/claroty_ctd-3ff35f11-fb3c-40b1-bb47-a0d5345fc0c7)\n\n[Insights](/app/dashboards#/view/claroty_ctd-e2106b06-3d16-4014-9343-f63ebb90a114)\n\n[Health Check](/app/dashboards#/view/claroty_ctd-b7e0ad66-86c3-4b72-ae19-0576ed9fb1d6)\n\n[Events](/app/dashboards#/view/claroty_ctd-66d8c2cc-eeb7-420e-8c83-9c0cae66a7fe)\n\n[Integration Page](/app/integrations/detail/claroty_ctd/overview)\n\n## Overview\n\nThe dashboard offers a concise view, highlighting total events, alerts, baseline figures, and a breakdown of events by type for quick insights.",`

I don't think that this will require a stack version bump.

**Comment by muskan-agarwal26** [comment]
@efd6 , as suggested, i have added double  new lines between the navigation links, also kept the stack version same as before, i.e, 8.13.0

File: packages/claroty_ctd/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: 0.1.3
```
Comment (kcreddy) [inline packages/claroty_ctd/changelog.yml:2]:
```suggestion
- version: 0.2.0
```

**Review by kcreddy — COMMENTED** [review]
Please add a dashboard screenshot (before and after) to the PR description.

**Review by kcreddy — APPROVED** [review]
LGTM. Can be merged after @efd6 approval.

**Comment by muskan-agarwal26** [comment]
Sure @kcreddy , will follow this method from next PR’s review comments.

**Review by efd6 — APPROVED** [review]

**Pull request merged**